src/worker/tasks/processEventLogsWorker.ts

```diff
@@ -54,13 +58,15 @@ const handler: Processor<any, void, string> = async (job: Job<string>) => {
   }
 
   // Any logs inserted in a delayed job indicates missed logs in the realtime job.
-  if (job.delay > 0) {
+  if (job.opts.delay && job.opts.delay > 0) {
```

Comment:
Why the first condition? Is opts.delay sometimes undefined?

Reply:
yeah, it can be `undefined`